.github/workflows/ci.yml

```diff
@@ -63,7 +66,7 @@ jobs:
         run: yarn --non-interactive --frozen-lockfile
 
       - name: Run tests
-        run: yarn test:coverage --maxWorkers=2 --workerIdleMemoryLimit=2000MB
+        run: yarn test:coverage --maxWorkers=2 --workerIdleMemoryLimit=2000MB --shard=${{ matrix.shard }}/${{ strategy.job-total }}
```

Comment:
LGTM, but it's not clear to me where `strategy.job-total` comes from, or how it is configured

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#strategy-context

This has some documentation about the strategy context. It's not _ideal_, because if we for some reason add another matrix, for example for NodeJS versions, we will need to come up with a different solution here. At that point, we may have to build a custom script to manage it because it's not clear to me that github actions has a way to get the length of a specific matrix array or anything.

Reply:
Ok, came up with a slightly less brittle solution - defining the matrix jobs as '1/3', '2/3', '3/3', etc. It doesn't feel great but it's less likely to randomly blow up in the future.